modules/transport-netty4/src/javaRestTest/java/org/elasticsearch/rest/Netty4HeadBodyIsEmptyIT.java

```diff
@@ -131,8 +131,8 @@ public void testTemplateExists() throws IOException {
             if (inFipsJvm()) {
```

Comment:
I missed it when I added the check originally but I had to spend 15 mins again now to figure out why we get warnings only in FIPS mode. Would you mind adding a comment saying that we need to expect warnings because in FIPS 140 mode we run with default distribution instead of integ-test distribution and this triggers these warnings?

Reply:
I added the comment. But this PR is for 7.12 only. I'll push the same comment separately to master and 7.x